No section in debian control

`Section` must be filled to include in apt repository:

```
# reprepro includedeb sid radical-native_v0.1beta15_x86-amd64.deb
No section given for 'radical-native', skipping.
```

https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-section
https://www.debian.org/doc/debian-policy/ch-archive.html#s-subsections

Looks like #22 depends on this issue.

<details>
<summary>I've even installed nodejs 14.5.0 from deb.nodesource.com, but `npm run build:native` fails the same as 12.18.1 from debian</summary>

```
   Compiling keytar-sys v0.1.1 (https://github.com/stoically/keytar-rs?rev=2dffd9f#2dffd9f7)
error: failed to run custom build command for `keytar-sys v0.1.1 (https://github.com/stoically/keytar-rs?rev=2dffd9f#2dffd9f7)`

Caused by:
  process didn't exit successfully: `/tmp/radical-native/target/debug/build/keytar-sys-27725ba0791125d7/build-script-build` (exit code: 101)
--- stderr
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Failure { command: "\"pkg-config\" \"--libs\" \"--cflags\" \"libsecret-1\"", output: Output { status: ExitStatus(ExitStatus(256)), stdout: "", stderr: "Package libsecret-1 was not found in the pkg-config search path.\nPerhaps you should add the directory containing `libsecret-1.pc\'\nto the PKG_CONFIG_PATH environment variable\nNo package \'libsecret-1\' found\n" } }', /home/sergio/.cargo/git/checkouts/keytar-rs-a0c7fcf28704269c/2dffd9f/keytar-sys/build.rs:15:23
stack backtrace:
   0: backtrace::backtrace::libunwind::trace
             at /usr/src/rustc-1.43.0/vendor/backtrace/src/backtrace/libunwind.rs:86
   1: backtrace::backtrace::trace_unsynchronized
             at /usr/src/rustc-1.43.0/vendor/backtrace/src/backtrace/mod.rs:66
   2: std::sys_common::backtrace::_print_fmt
             at src/libstd/sys_common/backtrace.rs:78
   3: <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt
             at src/libstd/sys_common/backtrace.rs:59
   4: core::fmt::write
             at src/libcore/fmt/mod.rs:1063
   5: std::io::Write::write_fmt
             at src/libstd/io/mod.rs:1426
   6: std::sys_common::backtrace::_print
             at src/libstd/sys_common/backtrace.rs:62
   7: std::sys_common::backtrace::print
             at src/libstd/sys_common/backtrace.rs:49
   8: std::panicking::default_hook::{{closure}}
             at src/libstd/panicking.rs:204
   9: std::panicking::default_hook
             at src/libstd/panicking.rs:224
  10: std::panicking::rust_panic_with_hook
             at src/libstd/panicking.rs:470
  11: rust_begin_unwind
             at src/libstd/panicking.rs:378
  12: core::panicking::panic_fmt
             at src/libcore/panicking.rs:85
  13: core::result::unwrap_failed
             at src/libcore/result.rs:1222
  14: core::result::Result<T,E>::unwrap
             at /usr/src/rustc-1.43.0/src/libcore/result.rs:1003
  15: build_script_build::main
             at ./build.rs:15
  16: std::rt::lang_start::{{closure}}
             at /usr/src/rustc-1.43.0/src/libstd/rt.rs:67
  17: std::rt::lang_start_internal::{{closure}}
             at src/libstd/rt.rs:52
  18: std::panicking::try::do_call
             at src/libstd/panicking.rs:303
  19: __rust_maybe_catch_panic
             at src/libpanic_unwind/lib.rs:86
  20: std::panicking::try
             at src/libstd/panicking.rs:281
  21: std::panic::catch_unwind
             at src/libstd/panic.rs:394
  22: std::rt::lang_start_internal
             at src/libstd/rt.rs:51
  23: std::rt::lang_start
             at /usr/src/rustc-1.43.0/src/libstd/rt.rs:67
  24: main
  25: __libc_start_main
  26: _start
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.

warning: build failed, waiting for other jobs to finish...
error: build failed
npm ERR! code ELIFECYCLE
npm ERR! errno 101
npm ERR! radical-native@ build:native: `cross-env ./native/scripts/build.sh`
npm ERR! Exit status 101
npm ERR! 
npm ERR! Failed at the radical-native@ build:native script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/sergio/.npm/_logs/2020-07-10T20_27_10_728Z-debug.log
```
</details>

<details>
<summary>/home/sergio/.npm/_logs/2020-07-10T20_27_10_728Z-debug.log</summary>

```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/bin/node', '/usr/bin/npm', 'run', 'build:native' ]
2 info using npm@6.14.5
3 info using node@v14.5.0
4 verbose run-script [ 'prebuild:native', 'build:native', 'postbuild:native' ]
5 info lifecycle radical-native@~prebuild:native: radical-native@
6 info lifecycle radical-native@~build:native: radical-native@
7 verbose lifecycle radical-native@~build:native: unsafe-perm in lifecycle true
8 verbose lifecycle radical-native@~build:native: PATH: /usr/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/tmp/radical-native/node_modules/.bin:/home/sergio/local/bin:/home/sergio/.local/bin:/home/sergio/Applications/.bin:/home/sergio/local/bin:/home/sergio/.local/bin:/usr/local/bin:/usr/bin:/bin:/usr/games
9 verbose lifecycle radical-native@~build:native: CWD: /tmp/radical-native
10 silly lifecycle radical-native@~build:native: Args: [ '-c', 'cross-env ./native/scripts/build.sh' ]
11 silly lifecycle radical-native@~build:native: Returned: code: 101  signal: null
12 info lifecycle radical-native@~build:native: Failed to exec build:native script
13 verbose stack Error: radical-native@ build:native: `cross-env ./native/scripts/build.sh`
13 verbose stack Exit status 101
13 verbose stack     at EventEmitter.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
13 verbose stack     at EventEmitter.emit (events.js:314:20)
13 verbose stack     at ChildProcess.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:314:20)
13 verbose stack     at maybeClose (internal/child_process.js:1051:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:287:5)
14 verbose pkgid radical-native@
15 verbose cwd /tmp/radical-native
16 verbose Linux 5.7.0-1-amd64
17 verbose argv "/usr/bin/node" "/usr/bin/npm" "run" "build:native"
18 verbose node v14.5.0
19 verbose npm  v6.14.5
20 error code ELIFECYCLE
21 error errno 101
22 error radical-native@ build:native: `cross-env ./native/scripts/build.sh`
22 error Exit status 101
23 error Failed at the radical-native@ build:native script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 101, true ]
```
</details>

--- a/native/Cargo.toml
+++ b/native/Cargo.toml
@@ -24,12 +24,13 @@ tempfile = "3.1.0"
 depends = "libsecret-1-0, libsqlcipher0"
 section = "utils"
 extended-description = """\
+This is a host application for Radical Native browser extension that enhances Riot Web with native capabilities.
+The communication is handled through Native Messaging API.
+.
 Features
     Search functionality in encrypted rooms using seshat
-    Secure OS key storage for pickle keys using keytar
-
-Supported Browsers
-    Firefox"""
+    Secure OS key storage for pickle keys using keytar\
+"""
 
 assets = [
     ["target/release/radical-native", "/usr/bin/", "755"],

--- a/native/Cargo.toml
+++ b/native/Cargo.toml
@@ -22,6 +22,15 @@ tempfile = "3.1.0"
 
 [package.metadata.deb]
 depends = "libsecret-1-0, libsqlcipher0"
+section = "utils"
+extended-description = """\
+Features
+    Search functionality in encrypted rooms using seshat
+    Secure OS key storage for pickle keys using keytar
+
+Supported Browsers
+    Firefox"""
+
 assets = [
     ["target/release/radical-native", "/usr/bin/", "755"],
     ["target/release/radical.native.json", "/usr/lib/mozilla/native-messaging-hosts/", "644"]

